### DISCUSSION TITLE
How can we set preview text for our emails that will show only in subject line not in the top of email body.

### DISCUSSION THREAD

**Discussion by Gurarkpreet** [aws/aws-sdk-go-v2#2555] [Q&A]
We were finding out to set preview text for our emails. We have tried many things like - setting preview_text and email_template in Body field of Message but it show preview text in the beginning of the email body. But we need to show preview text only in subject line not in the top of email body.
Is there any way to set preview_text only in subject line.

**Comment by lucix-aws** [comment]
As SDK owners we are generally not subject matter experts on specific service behavior or usage. I personally have not used the `ses` or `sesv2` service.

Questions of this nature are best resolved by using the AWS console to open a support case with the service.
